lms/templates/courseware/_program_type_faq.html

```diff
@@ -0,0 +1,50 @@
+<%page args="program_type" expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangolib.markup import Text
+%>
+<%namespace name='static' file='../static_content.html'/>
+
+<div class="accordion-item">
+  <div class="accordion-head">
+    ${Text(_('What is a {program_type}?')).format(program_type=program_type)}
+  </div>
+  <div class="accordion-content">
+    <p>
+      ${_("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum")}
```

Comment:
Since we were originally implementing this page in individual themes and at least initially each site would only have a single program type associated with it, the plan was to hard code the real content here once the partner settle on a description of the program type. Since we are moving this to edx-platform, we will need to find a new solution for populating this section.